package.json

```diff
@@ -15,6 +15,7 @@
     },
     "dependencies": {
         "@babel/preset-env": "^7.24.5",
+        "@chakra-ui/react": "^2.8.2",
```

Comment:
in deriv ui, will you also use charkra ui 👀

Reply:
No it's a temporary thing and a quick fix

Reply:
@niloofar-deriv can we have the fix in deriv-ui instead of installing this dependency?

Reply:
Base on my discussion with @matin-deriv @ali-hosseini-deriv we decided to unblock the release for now in parallel I'm working on fixing this issue from ui side as well

Reply:
@farrah-deriv we will make sure to keep the api the same when we fix it later in ui so there won't be any changes from ur side other than changing the import path.